.github/workflows/deploy-prod.yaml

```diff
@@ -17,119 +17,117 @@ jobs:
         export VERSION_TAG=$GITHUB_REF_NAME
         ./bin/verify-tag.sh "${VERSION_TAG}"
 
-  generate-kurl-release-notes-pr:
+  get-staging-release:
     runs-on: ubuntu-20.04
     needs:
     - verify-tag
+    outputs:
+      staging_release: ${{ steps.get_staging_release.outputs.staging_release }}
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-
-    - name: Generate kURL Release Notes PR
-      env:
-        GH_PAT: ${{ secrets.AUTOMATED_PR_GH_PAT }}
+    - name: find corresponding staging release for tag
+      id: get_staging_release
       run: |
-        export VERSION_TAG=$GITHUB_REF_NAME
-        curl -H "Authorization: token $GH_PAT" \
-          -H 'Accept: application/json' \
-          -d "{\"event_type\": \"kubernetes-installer-release-notes\", \"client_payload\": {\"version\": \"${VERSION_TAG}\" }}" \
-          "https://api.github.com/repos/replicatedhq/replicated-docs/dispatches"
+        version_commit="$(git rev-list "${GITHUB_REF_NAME}" -n 1 | xargs git rev-parse --short)"
+        staging_release_dir="$(aws s3 ls s3://"${S3_BUCKET}/${STAGING_PREFIX}/" | grep "${version_commit}" | awk '{print $2}')"
+        if [ -z "${staging_release_dir}" ]; then
+            echo "Could not find staging release for tag ${GITHUB_REF_NAME} (${version_commit})"
+            return 1
```

Comment:
can you return outside of a function? should this be exit 1?

Reply:
Got my wires crossed! Will fix this.